maintenance-branch

```diff
@@ -1 +1 @@
-4-0-dev
\ No newline at end of file
+drop-old-rubies
```

Comment:
⚠️ This change is temporary and needs to be removed before merging, or soon after ⚠️ 

This is due to changes from sibling PRs being inter-dependent, e.g. a removed `jruby_9000?` method is still in use in `rspec-core` on branch 4-0-dev, but all its usages are removed on branch drop-old-rubies.

Reply:
We'll be merging this PR last, so its should be safe to drop this.

Reply:
I can't drop this one - other repos' 4-0-dev branches still use methods that we've removed here (`jruby_version` is one example 😉 ). If we run their corresponding build steps, their specs will fail.

Let's remove this change once we've merged the rest of the repos?

Reply:
Other repos are merged, do you want to drop this change?